doc/contributing/documentation_guide.md

```diff
@@ -164,6 +164,10 @@ We might consider whether to suppress when:
   (e.g., _Array_ in the documentation for class `Array`).
 - The same reference is repeated many times
   (e.g., _RDoc_ on this page).
+- The reference is to a class or module that users
+  usually don't deal with, including these:
+
+  - \Method.
```

Comment:
I think we can also include `Class` and `Module` in this list.

Reply:
Added.